test/input.txt

```diff
@@ -147,6 +147,11 @@
   # clears all
   clear
   list
+  add Betsy Choo pp/222222 pe/benchoo@nus.edu.sg pa/222, beta street t/secretive
+  add Adam Brown p/111111 e/adam@gmail.com a/111, alpha street
+  list
+  sort
```

Comment:
Good job in adding a test case for the IO, maybe you can start trying out the JUnit test suite to familiarise yourself with it.